drivers/bridge/bridge.go

```diff
@@ -145,6 +146,159 @@ func (c *NetworkConfiguration) Validate() error {
 	return nil
 }
 
+// FromMap retrieve the configuration data from the map form.
+func (c *networkConfiguration) FromMap(data map[string]interface{}) error {
```

Comment:
Agree with @mavenugo , I can restore the gnerateFrommodel along with the new one.